62.CAD/common.hlsl

```diff
@@ -22,10 +24,53 @@ struct DrawObject
     uint64_t geometryAddress;
 };
 
+template <typename float_t>
+struct QuadBezierAnalyticArcLengthCalculator
```

Comment:
remove `ArcLengthPrecomputedValues`, and move this `QuadBezierAnalyticArcLengthCalculator` to `beziers.hlsl`

`calcArcLen` should be a method of this struct now

Reply:
> rename this to `AnalyticArcLengthCalculator`
`QuadraticAnalyticArcLengthCalculator`? after all its only for quadratic functions

Reply:
done